lib/FreeRTOS-Plus-TCP/source/FreeRTOS_DNS.c

```diff
@@ -420,6 +420,8 @@ uint32_t FreeRTOS_gethostbyname_a( const char *pcHostName, FOnDNSEvent pCallback
 uint32_t ulIPAddress = 0UL;
 TickType_t xReadTimeOut_ms = ipconfigSOCK_DEFAULT_RECEIVE_BLOCK_TIME;
 TickType_t xIdentifier = 0;
+	configASSERT(pcHostName); /* pcHostName is not NULL */
```

Comment:
Please remove the whitespace at the beginning of the line. Also the comment seems redundant, please remove.